sessionctx/variable/sysvar.go

```diff
@@ -1857,6 +1857,10 @@ var defaultSysVars = []*SysVar{
 		s.EnablePseudoForOutdatedStats = TiDBOptOn(val)
 		return nil
 	}},
+	{Scope: ScopeGlobal | ScopeSession, Name: TiDBRegardNULLAsPoint, Value: BoolToOnOff(DefTiDBRegardNULLAsPoint), Type: TypeBool, SetSession: func(s *SessionVars, val string) error {
+		s.StmtCtx.RegardNULLAsPoint = TiDBOptOn(val)
```

Comment:
According to `ResetContextOfStmt()`, which will be called in `ExecuteStmt()`, there will be a new `StmtCtx` every time when you execute a statement.
https://github.com/pingcap/tidb/blob/78c653e294e7fe464003498981ae3b6efa8b5cfb/executor/executor.go#L1855
Seems we should put the option in `SessionVars`.